api/src/main/java/org/apache/iceberg/ManifestFile.java

```diff
@@ -52,11 +52,11 @@ public interface ManifestFile {
       optional(
           503, "added_snapshot_id", Types.LongType.get(), "Snapshot ID that added the manifest");
   Types.NestedField ADDED_FILES_COUNT =
-      optional(504, "added_data_files_count", Types.IntegerType.get(), "Added entry count");
+      optional(504, "added_files_count", Types.IntegerType.get(), "Added entry count");
   Types.NestedField EXISTING_FILES_COUNT =
-      optional(505, "existing_data_files_count", Types.IntegerType.get(), "Existing entry count");
+      optional(505, "existing_files_count", Types.IntegerType.get(), "Existing entry count");
   Types.NestedField DELETED_FILES_COUNT =
-      optional(506, "deleted_data_files_count", Types.IntegerType.get(), "Deleted entry count");
+      optional(506, "deleted_files_count", Types.IntegerType.get(), "Deleted entry count");
```

Comment:
I thought we change the spec document instead of changing the schema (field names) here.
I saw the comments down below. Agree that this doesn't break compatibility. 

But If we follow this path, `manifest_entry` has a field called `data_file` which represents both data and delete file (https://github.com/apache/iceberg/blame/master/format/spec.md#L421) May be we need to rename that aswell?

Reply:
Hey @ajantha-bhat Thanks, that's actually a good point. However, If we go that route, I'd prefer to do that separately because of two reasons:
- Looking at the code, I also see that we occasionally fetch the struct by name.
- The code and spec are consistent, so we have to change them both

Reply:
ok. Sounds reasonable to me.